stl/inc/xtree

```diff
@@ -1321,14 +1321,27 @@ public:
     }
 
     size_type erase(const key_type& _Keyval) noexcept(noexcept(_Eqrange(_Keyval))) /* strengthened */ {
-        const auto _Where = _Eqrange(_Keyval);
+        return _Erase(_Eqrange(_Keyval));
+    }
+
+private:

```

Comment:
again with the new private section. We could just as well put _Erase up with the other private erase helpers